{"name": "test_login_noExistUser_wrongPasswd[data1]", "status": "passed", "attachments": [{"name": "log", "source": "794e4f1b-d701-45ec-990e-ee07ce162888-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'用户名': 'dsadsadsa', '密码': 'pytho', '预期': '请输入至少6位数的密码'}"}], "start": 1587375035126, "stop": 1587375036039, "uuid": "8e3cbad0-3561-4c95-b48e-5112c65f8cc5", "historyId": "634fa2f876868d6c8f81ec03e35c966f", "testCaseId": "d35b8259a99be3473adfcaff1df8682d", "fullName": "TestCases.test_00_login.TestLogin#test_login_noExistUser_wrongPasswd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "16528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_00_login"}]}